install.sh

```diff
@@ -4,23 +4,25 @@
 
 set -e
 
-case $(uname -s) in
+case "$(uname -s)" in
 Darwin) target="x86_64-apple-darwin" ;;
 *) target="x86_64-unknown-linux-gnu" ;;
 esac
 
-if [ $(uname -m) != "x86_64" ]; then
+if [ "$(uname -m)" != "x86_64" ]; then
 	echo "Unsupported architecture $(uname -m). Only x64 binaries are available."
 	exit
 fi
 
-if [ $# -eq 0 ]; then
-	deno_asset_path=$(
+if [ "$#" -eq 0 ]; then
+	deno_asset_path="$(
 		command curl -sSf https://github.com/denoland/deno/releases |
 			command grep -o "/denoland/deno/releases/download/.*/deno-${target}\\.zip" |
 			command head -n 1
-	)
-	if [ ! "$deno_asset_path" ]; then exit 1; fi
+	)"
+	if [ ! "$deno_asset_path" ]; then
+		exit 1
+	fi
```

Comment:
Formatting, please revert.

There's another PR which adresses error messages: https://github.com/denoland/deno_install/pull/122

If you want to improve error messages, feel free to add to that PR instead. :)

Reply:
This revert would make this PR an empty string :)